.github/workflows/cleanup-firewall-rules.yml

```diff
@@ -0,0 +1,94 @@
+name: Manual Firewall rules Deletion Preprod and Prouction
+
+on:
+  workflow_dispatch:
+  schedule: 
+    - cron: "0 2 * * 5"
```

Comment:
Instead of creating 2 jobs in 1 workflow, we should create a reusable action, and call it from 2 separate workflows (preprod and prod)